instrumentation/apache-pulsar/apache-pulsar-2.8/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/pulsar/v2_8/telemetry/PulsarSingletons.java

```diff
@@ -24,7 +24,8 @@
 import org.apache.pulsar.client.api.Message;
 
 public final class PulsarSingletons {
-  private static final String INSTRUMENTATION_NAME = "io.opentelemetry.pulsar-client-2.8";
```

Comment:
Instrumentation name has to be the same as the module name. If you guys feel that `pulsar-client-2.8` would be better I can rename the module in another pr.

Reply:
I opened #8004 as an alternative